config/locales/en.yml

```diff
@@ -105,14 +105,14 @@ en:
 
   # Bottom navigation
   bottom_navigation:
-    cancel: "Cancel"
-    continue: "Continue"
+    cancel: " ⟵ Cancel"
```

Comment:
Swapped out arrow .png files with unicode arrows.  Allows for better grouping during the hover state effect.  Loads faster than image file.